constraints.txt

```diff
@@ -0,0 +1,239 @@
+alabaster==0.7.12
+antlr4-python3-runtime==4.9.2
+argon2-cffi==21.1.0
+attrs==21.2.0
+Babel==2.9.1
+backcall==0.2.0
+backports.entry-points-selectable==1.1.0
+bcp47==0.0.4
+beautifulsoup4==4.10.0
+bidict==0.21.3
+black==21.8b0
+bleach==4.1.0
+bmt==0.7.4
+cachetools==4.2.2
+certifi==2021.5.30
+cffi==1.14.6
+CFGraph==0.2.1
+chardet==4.0.0
+charset-normalizer==2.0.4
+click==7.1.2
+colorama==0.4.4
+commonmark==0.9.1
+cryptography==3.4.8
+cycler==0.10.0
+dacite==1.6.0
+daff==1.3.46
+dataclasses==0.6
+dataclasses-json==0.5.5
+dataclasses-jsonschema==2.14.1
+debugpy==1.4.3
+decorator==5.1.0
+defusedxml==0.7.1
+Deprecated==1.2.13
+deprecation==2.1.0
+diskcache==5.2.1
+distlib==0.3.2
+docker==5.0.2
+docutils==0.16
+entrypoints==0.3
+filelock==3.0.12
+flake8==3.9.2
+frozendict==2.0.6
+funowl==0.1.9
+ghp-import==2.0.1
+google==3.0.0
+google-api-core==2.0.1
+google-api-python-client==2.20.0
+google-auth==2.0.2
+google-auth-httplib2==0.1.0
+google-auth-oauthlib==0.4.6
+googleapis-common-protos==1.53.0
+graphviz==0.17
+greenlet==1.1.1
+gspread==4.0.1
+gspread-formatting==1.0.4
+hbreader==0.9.1
+httplib2==0.19.1
+idna==3.2
+ijson==3.1.4
+imagesize==1.2.0
+importlib-metadata==4.8.1
+iniconfig==1.1.1
+ipykernel==6.4.1
+ipython==7.27.0
+ipython-genutils==0.2.0
+ipywidgets==7.6.4
+isodate==0.6.0
+jedi==0.18.0
+jeepney==0.7.1
+Jinja2==3.0.1
+joblib==1.0.1
+jsobject==0.10.2
+json-flattener==0.1.7
+jsonasobj==1.3.1
+jsonasobj2==1.0.4
+jsonlines==2.0.0
+jsonpath-rw==1.4.0
+jsonpickle==2.0.0
+jsonschema==3.2.0
+jsonstreams==0.6.0
+jupyter==1.0.0
+jupyter-client==7.0.2
+jupyter-console==6.4.0
+jupyter-core==4.7.1
+jupyterlab-pygments==0.1.2
+jupyterlab-widgets==1.0.1
+keyring==23.2.1
+kgx==1.5.2
+kiwisolver==1.3.2
+linkml==1.0.6
+linkml-model==1.0.4
+linkml-runtime==1.1.0
+lxml==4.6.3
+Markdown==3.3.4
+MarkupSafe==2.0.1
+marshmallow==3.13.0
+marshmallow-enum==1.5.1
+matplotlib==3.4.3
+matplotlib-inline==0.1.3
+matplotlib-venn==0.11.6
+mccabe==0.6.1
+mergedeep==1.3.4
+mistune==0.8.4
+mkdocs==1.2.2
+mypy==0.910
+mypy-extensions==0.4.3
+nbclient==0.5.4
+nbconvert==6.1.0
+nbformat==5.1.3
+neo4jrestclient==2.1.1
+nest-asyncio==1.5.1
+networkx==2.6.3
```

Comment:
Its odd that this is here, and requirements.txt has a fixed `networkx` version

Reply:
I thought I had removed all version numbers from the requirements.txt file… I must have done that in another branch and forgot to bring it to that branch… :facepalm:

Reply:
All the better then! I would like that, because the old networkx breaks several of my pipelines now.

Reply:
I'll push an updated `requirements.txt` later today. Just re-building now to check that everything still works with the new `requirements.txt`. (I don't expect any breakage but…)